modules/@apostrophecms/image/ui/apos/components/AposMediaManagerEditor.vue

```diff
@@ -10,7 +10,7 @@
         <img
           v-if="activeMedia.attachment && activeMedia.attachment._urls"
           class="apos-media-editor__thumb"
-          :src="activeMedia.attachment._urls['one-third']" :alt="activeMedia.description"
+          :src="activeMedia.attachment._urls[wasInTrash ? 'one-sixth' : 'one-third']" :alt="activeMedia.description"
```

Comment:
Longstanding thing: when in the trash only `one-sixth` is still readable, as a compromise to allow editors to be sure they chose the right thing to restore.